collections/billiongraves.js

```diff
@@ -115,6 +136,25 @@ function parseBillionGraves(htmlstring, familymembers, relation) {
     return profiledata;
 }
 
+function getData(htmlstring) {
+    return JSON.parse(htmlstring.split('<script type="application/ld+json">')[1].split('</script>')[0]);
```

Comment:
@eljeffeg this is the part I really dislike. I couldn't fetch <script> content via jQuery, so try to take a look if it's possible.